src/index.html

```diff
@@ -7,8 +7,30 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo"><img src="./images/logo.png" alt="MOYO logo"></a>
+      <nav class="nav_bar">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="link is-active" href="#">Apple</a></li>
+          <li class="nav__item"><a class="link" href="#">Samsung</a></li>
+          <li class="nav__item"><a class="link" href="#">Smartphones</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a class="nav__link" href="#">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Smartphones</a>
          </li>
```
As I mentioned previously links are related to the `nav` too, so `nav__link` is a more suitable name in this case
Checklist:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/26f3f704-d077-4e2c-8269-936f3ed596b5)